src/components/TipInput.vue

```diff
@@ -136,12 +137,12 @@ export default {
           token: null,
         }
         : {
-          value: this.largestFtTipAmount.amount,
-          token: this.largestFtTipAmount.token,
+          value: this.largestFtTipAmount ? this.largestFtTipAmount.amount : null,
```

Comment:
```suggestion
          value: this.largestFtTipAmount?.amount,
```
If there is no particular reason to have null in value, we can do this.

Reply:
I think there is, at least it didn't work for me without